2-mandatory/4-restaurant.js

```diff
@@ -20,8 +20,23 @@ const MENU = {
 };
 
 let cashRegister = {
-  // write code here
-}
+  orderBurger: function (balance) {
+    let balanceAfterBurger = 0;
+    if (balance >= MENU.burger) {
+      return balance - MENU.burger;
+    } else {
+      return balance + " Transaction denied, insufficient balance";
```

Comment:
This is almost perfect! However, when you run the test - you'll see:
```
Expected: 6.49
Received: "6.49 Transaction denied, insufficient balance"
```
Can you think of how to fix this?